### DISCUSSION TITLE
How to setup relationships in PocketBase?

### DISCUSSION THREAD

**Discussion by timomartinson** [pocketbase/pocketbase#3101] [Q&A]
I don't know if this question is already answered. So I just ask here.

How to create many-to-many and one-to-many relationships in PocketBase?

Thank you. Timo.

**Comment by ganigeorgiev** [comment]
The question is too broad and I'm not sure what you are looking for and how to answer it.

In general, you could check https://pocketbase.io/docs/working-with-relations/.

Managing relations in PocketBase is not too different than in any other relational database.
The only thing PocketBase adds in addition is the support of a multiple relations without an extra junction table but you don't have to use it if you don't want your relation field to hold more than one value.

**Reply by ganigeorgiev** [reply]
_Since the above link doesn't include a direct/straightforward example for many-to-many relation - you can use a junction collection for this use case. For example, if you want a many-to-many relation between `users` and `posts` collections, let's say for storing likes, you can create a 3rd collection `userPostLikes` that could have 2 single `relation` fields pointing to the `users` and `posts` collection. The `comments` collection in the above link is also such example._
